rivet.spec

```diff
@@ -1,7 +1,8 @@
-### RPM external rivet 2.4.0
+### RPM external rivet 2.5.2
 Source: http://cern.ch/service-spi/external/MCGenerators/distribution/rivet/rivet-%{realversion}-src.tgz
+#Source: http://www.hepforge.org/archive/rivet/Rivet-%{realversion}.tar.gz  
 
-Requires: hepmc boost fastjet gsl yaml-cpp yoda
+Requires: hepmc fastjet gsl yoda
```

Comment:
You can look around for examples (e.g. https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_8_1_X/gcc530/geant4-toolfile.spec) GEANT4 uses:
```
  <use name="clhep"/>
  <use name="xerces-c"/>
```